### PR TITLE
FIP-0052: First draft of FIP proposing increased max sector commitment duration

### PR DESCRIPTION
**Pull request by anorth**
FIP-0047 introduces a mechanism to decouple a sector’s PoRep validity from its commitment duration. After this, we are free to increase the maximum sector commitment duration from the current 1.5 years up to any value below 5 years (the maximum sector total lifetime).

This is a proposal to increase the maximum commitment duration, motivated primarily by the product benefits. This proposal does not include a multiplier or other direct incentive to make longer commitments, but does not prevent such incentives being added later. Other values derived from the maximum sector commitment duration, such as the built-in market actor's maximum deal duration, are increased in line.

Discussion: https://github.com/filecoin-project/FIPs/discussions/475

### CODE REVIEW

File: FIPS/fip-xxxx-sector-commitment-duration.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+type: Technical (Core)
+category Core
+created: 2022-12-06
+requires: FIP-0047
+---
+## Simple Summary
+Increase the maximum sector commitment duration from 540 days to 1278 days (3.5 years).
+
+## Abstract
+FIP-0047 introduces a mechanism to decouple a sector’s PoRep validity from its commitment duration.
```
Comment (jennijuju) [inline FIPS/fip-xxxx-sector-commitment-duration.md:16]:
```suggestion
[FIP-0047](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0047.md) introduces a mechanism to decouple a sector’s PoRep validity from its commitment duration.
```

File: FIPS/fip-xxxx-sector-commitment-duration.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+After this, we are free to increase the maximum sector commitment duration from the current 1.5 years
+up to any value below 5 years (the maximum sector total lifetime).
```
Comment (jennijuju) [inline FIPS/fip-xxxx-sector-commitment-duration.md:18]:
```suggestion
up to any value below 5 years (the maximum sector total lifetime/current PoRep security lifetime).
```

File: FIPS/fip-xxxx-sector-commitment-duration.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+This is a proposal to increase the maximum commitment duration, motivated primarily by the product benefits.
+This proposal does not include a multiplier or other direct incentive to make longer commitments,
+but does not prevent such incentives being added later.
```
Comment (jennijuju) [inline FIPS/fip-xxxx-sector-commitment-duration.md:22]:
```suggestion

```

I dont know if this call out is necessary in the context of this FIP as a standalone. i.e: will you still call this out without the fip0036 context?

File: FIPS/fip-xxxx-sector-commitment-duration.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+up to a limit on the sector's total active life of 5 years.
+This limit (coupled with current limitations of sector storage and markets) in turn prevents a provider
+from committing to a deal any longer than 1.5 years.
+
+There is significant client demand for longer deal terms.
+Longer commitments will also contribute to both the stability of the storage-powered consensus base,
+and decrease the velocity of tokens that are pledged to those sectors.
+
+## Specification
+- Set the built-in miner actor's maximum sector expiration extension to 1278 days.
```
Comment (jennijuju) [inline FIPS/fip-xxxx-sector-commitment-duration.md:38]:
```suggestion
- Set the built-in miner actor's [maximum v1-proof sector expiration ](https://github.com/filecoin-project/builtin-actors/blob/dc8077c30694bda58559cd057c6d1a0e1b73fc6e/actors/miner/src/policy.rs#L79) extension to 1278 days.
```

File: FIPS/fip-xxxx-sector-commitment-duration.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+This value bounds the initial commitment duration as well as any subsequent extension.
+- Set the built-in storage market actor's maximum deal duration to 1278 days.
```
Comment (jennijuju) [inline FIPS/fip-xxxx-sector-commitment-duration.md:40]:
```suggestion
- Set the built-in storage market actor's[ maximum deal duration](https://github.com/filecoin-project/builtin-actors/blob/dc8077c30694bda58559cd057c6d1a0e1b73fc6e/actors/market/src/policy.rs#L22) to 1278 days.
```

File: FIPS/fip-xxxx-sector-commitment-duration.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+All sectors, including those already committed, are eligible for extension up by to 1278 days (subject to the 5-year maximum lifetime).
```
Comment (jennijuju) [inline FIPS/fip-xxxx-sector-commitment-duration.md:42]:
```suggestion
All sectors, including those already committed, are eligible for  a single extension up by to 1278 days (subject to a total 5-year maximum lifetime).
```

File: FIPS/fip-xxxx-sector-commitment-duration.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+## Design Rationale
+This proposal was borne out of the decoupling of proof validity duration from sector commitment duration in
+[FIP-0047](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0047.md), itself initially created 
+to support [FIP-0036](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0036.md).
+This proposal is intended to be the simplest and least-controversial change possible to enable the 
+product utility of longer deals, with minimal direct crypto-economic impact.
+
+This proposal does not exclude the addition of incentives, such as a duration multiplier, by future FIPs.
```
Comment (jennijuju) [inline FIPS/fip-xxxx-sector-commitment-duration.md:51]: have a similar q as above

File: FIPS/fip-xxxx-sector-commitment-duration.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+face the possibility of sector termination if they fail to post the proof refresh message on schedule.
+
+## Incentive Considerations
+<!--All FIPs must contain a section that discusses the incentive implications/considerations relative to the proposed change. Include information that might be important for incentive discussion. A discussion on how the proposed change will incentivize reliable and useful storage is required. FIP submissions missing the "Incentive Considerations" section will be rejected. An FIP cannot proceed to status "Final" without a Incentive Considerations discussion deemed sufficient by the reviewers.-->
+All FIPs must contain a section that discusses the incentive implications/considerations relative to the proposed change. Include information that might be important for incentive discussion. A discussion on how the proposed change will incentivize reliable and useful storage is required. FIP submissions missing the "Incentive Considerations" section will be rejected. An FIP cannot proceed to status "Final" without a Incentive Considerations discussion deemed sufficient by the reviewers.
```
Comment (jennijuju) [inline FIPS/fip-xxxx-sector-commitment-duration.md:84]:
```suggestion

```

File: FIPS/fip-xxxx-sector-commitment-duration.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+
+When the per-sector initial pledge amount is increasing (as is the case when net onboarding falls behind net token emissions),
+committed-capacity providers can lock-in the current, lower, pledge amount for a longer term
+rather than face a decision between re-pledging a higher amount when extending their incremental commitment,
+or letting the sector expire.
+A longer commitment has the small beneficial effect to the network of locking that pledge for longer, delaying future emissions.
+
+## Product Considerations
+The primary impact of this proposal is on deal clients and the storage providers serving them.
+Supporting a longer deal minimum-term allows clients with long-term archival data to secure storage for their data for that term.
```
Comment (jennijuju) [inline FIPS/fip-xxxx-sector-commitment-duration.md:99]:
```suggestion
Supporting a longer deal maximum-term allows clients with long-term archival data to secure storage for their data for that term.
```

File: FIPS/fip-xxxx-sector-commitment-duration.md (hunk 3 above)
Comment (anorth) [reply FIPS/fip-xxxx-sector-commitment-duration.md:22]: I understand the comment, but given that FIP-0036 and follow-ups are in fact a context, I think it's worth explicitly pointing out the difference here. Participants in discussion thread didn't automatically see this.

File: FIPS/fip-xxxx-sector-commitment-duration.md (hunk 6 above)
Comment (anorth) [reply FIPS/fip-xxxx-sector-commitment-duration.md:42]: Adding "a single" is incorrect, if it implies that a sector can only be extended once. The lifetime limitation is already described. A  sector can be extended multiple times subject to that lifetime restriction.

Comment (jennijuju) [reply FIPS/fip-xxxx-sector-commitment-duration.md:45]: yah that was misleading. I was hoping to be more clear about
- each extention can be up to 1278 days
- multiple extensions are allowed subject to the 5 year maximum lifetime

File: FIPS/fip-xxxx-sector-commitment-duration.md [hunk 10]
```diff
@@ -0,0 +1,126 @@
+---
+fip: "<to be assigned>"
```
Comment (jennijuju) [inline FIPS/fip-xxxx-sector-commitment-duration.md:2]: fip-0052

**Pull request merged**